src/toast/examples.js

```diff
@@ -7,8 +7,9 @@ LICENSE file in the root directory of this source tree.
 // @flow
 import * as React from 'react';
 import {styled} from '../styles';
-import {Toast, KIND} from './index';
+import {Toast, toaster, KIND, PLACEMENT} from './index';
```

Comment:
should we use `Toaster` to be consistent with `Toast`?

Reply:
There is a `Toaster` component, but the lower case one is different. The lower cased one renders the `Toaster` on `create`.